evm/src/cpu/decode.rs

```diff
@@ -160,6 +163,10 @@ pub fn eval_packed_generic<P: PackedField>(
 ) {
     let cycle_filter = lv[columns::IS_CPU_CYCLE];
 
+    // If cycle_filter = 0, then we require is_eq = 0 and is_iszero = 0.
```

Comment:
I'm not sure this is needed? If `cycle_filter = 0`, it seems like we could leave them unconstrained since they'd be unused

Reply:
They are used though: I have some degree-2 constraints, so the filter must be degree-1. I can’t filter by, e.g., `cycle_filter * is_eq` and must filter by just `is_eq` instead. That means that `is_eq` must always be `0` if we’re not executing a cycle. (`is_iszero` is analogous.)